projects/cdk/schematics/ng-add/schema.json

```diff
@@ -11,6 +11,15 @@
                 "$source": "projectName"
             }
         },
+        "addGlobalStyles": {
+            "description": "Setting up global styles",
+            "type": "boolean",
+            "default": false,
+            "x-prompt": {
+                "message": "Do you want to use global styles for Taiga UI?",
```

Comment:
`global Taiga UI classes, such as 'tui-space', 'tui-skeleton', etc?` so that people know what it's about

Reply:
fixed